docs/sdk/server-side-sdks/go/go-usage.md

```diff
@@ -144,3 +144,23 @@ making bucketing decisions.
 In this example, the `Variable` call would make an API request and send along the user data specified by the call.
 That data would be used in combination with EdgeDB data to make correct bucketing decisions before returning the
 variable's value.
+
+
+## Enabling Beta Realtime Updates
+:::warning
+This feature is in beta, and may not function as expected. Please ensure that you have the latest version of the SDK.
+:::
+
+To enable Beta Realtime Updates, pass in the `EnableBetaRealtimeUpdates` option to the SDK initialization:
```

Comment:
Can you swap the order here, explain what the feature is, then show how to enable it.

Reply:
The order of the warning and all that? or just what the feature is portion?